.github/workflows/build_docs.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.7", "3.8", "3.9"]
+        python_version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
Not sure we can actually do this since 3.10 isn't supported by koalas and we use koalas in our docs. Woodwork only runs the docs build for 3.8 in its CI. Is it better to match Woodwork and only run on 3.8 or have all but 3.10 be run?

Reply:
Have all but 3.10 run (since its not slowing us down, and increase our testing scenarios).

Reply:
Do we need to put a note somewhere in the docs that 3.10 isn't support for Koalas (yet)?

Reply:
That also means the 3.10 unit tests koalas run is pretty much useless

Can we look into excluding them from the test runs?

Not exactly what we want but gives an idea:
https://github.community/t/how-to-conditionally-include-exclude-items-in-matrix-eg-based-on-branch/16853/6

Reply:
Seems like we would want to add 

```
      exclude:
        - python_version: "3.10"
        - libraries: "koalas"
```

Reply:
90a255c2b63349fb7e7c39902a3eefb6c47209d7

oops

```
[Invalid workflow file: .github/workflows/unit_tests_with_latest_deps.yml#L18](https://github.com/alteryx/featuretools/actions/runs/1959930281/workflow)
The workflow is not valid. .github/workflows/unit_tests_with_latest_deps.yml (Line: 18, Col: 7): Unexpected value 'exclude'
```

Reply:
I think it would be

```
exclude:
  - python_version: "3.10"
    libraries: "koalas"
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix

Reply:
using a dash mark for each would exclude all 3.10, and all koalas

Reply:
that is indeed what happened 🤦‍♀️  thank you!